from __future__ import annotations


def pytest_runtest_setup():
    raise Exception("2")
